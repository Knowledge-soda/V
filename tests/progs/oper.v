print(+(2, 12))
print(*(7, 11))
print(-(11, 2))
print(/(70, 5))
print(%(25, 3))
