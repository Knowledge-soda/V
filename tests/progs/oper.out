14
77
9
14
1
